src/test/data/DemoData.js

```diff
@@ -1,680 +1,534 @@
-const dataList = [
+export const dataList = [
```

Comment:
We may wish to change this const's name to `resultsList` to distinguish it now.

Reply:
I agree